from log.py import log_error


te = 1
log_error(te)


